### ISSUE TITLE
Main GUI window disappears off screen after minimized

### ISSUE THREAD

**Issue by bryc** [arnaud-neny/rePlayer#27]
In Windows 10, I use the "Minimize All" button at the bottom right corner, which basically shows desktop by minimizing all windows. This causes issues with rePlayer.

The biggest problem is that the main window (with play/stop buttons) changes its position, sometimes going off screen. So it cannot be reached again.

There is almost no way to fix this. The only workaround is to open Task Manager, and force "maximize" on rePlayer. But this behaves strangely and cannot be undone or resized to be small again.

A lesser issue of "Minimize All" is that it causes rePlayer to needlessly flash orange, which is a bit annoying.

Another separate concern is that, if I use the multiple desktop feature of Win10, and attempt to move rePlayer to a separate desktop, the playlist is shown on all desktops, but the main window is not.  This might have to do with rePlayer having a taskbar icon. Because with XMPlay, it doesn't support multiple desktops, and the taskbar icon is optional (I turn it off and access it via status icon tray). An option to hide the taskbar icon (and only access rePlayer via tray) might be useful.

**Comment by arnaud-neny** [comment]
Sadly I can't reproduce these issues. But so far, if the windows are changing positions, it's a Windows issues (the os is handling this).

**Comment by bryc** [comment]
Yeah I can't get a consistent reproduction case, sometimes it moves the main window to the very left of the screen, half cut off, sometimes, the window is gone.  And sometimes it just never does it when I'm deliberately trying to do it and recording. 😕 

I did manage to capture the window disappearing once here:


https://github.com/user-attachments/assets/1083a01a-63e4-4370-a01e-84d5fd47ab78

In video I click the bottom right button, the minimize all (aka show desktop) button, over and over again.  A few times clicking it twice doesn't bring rePlayer back, and have to click the flashing taskbar icon instead to get it back.

Essentially, the 'show desktop' button at bottom right comes with a random risk of the main window moving off screen somewhere, and impossible to get it back reasonably without closing and re-opening program.
